ecosystem/platform/server/app/jobs/kyc_complete_job.rb

```diff
@@ -3,16 +3,15 @@
 # Copyright (c) Aptos
 # SPDX-License-Identifier: Apache-2.0
 
-class KYCCompleteJobError < StandardError
-end
+class KYCCompleteJobError < StandardError; end
 
 class KYCCompleteJob < ApplicationJob
   # Ex args: { user_id: 32, inquiry_id=inq_syMMVRdEz7fswAa2hi }
   def perform(args)
     user = User.find(args[:user_id])
     sentry_scope.set_user(id: user.id)
+
     inquiry_id = args[:inquiry_id]
-    sentry_scope.set_context(:inquiry_id, inquiry_id)
```

Comment:
No longer needed?

Reply:
ApplicationJob now automatically add context for all job args :-)